packages/webpack/src/loader.ts

```diff
@@ -49,18 +45,25 @@ function createIdentifier(
   return `./${fileIdent}-${lang}-virtual.json!=!${unloader}!`;
 }
 
-const renderLanguageLoaderAsync =
-  (resourcePath: string, loadedTranslation: LoadedTranslation) =>
-  (lang: string) => {
-    const identifier = createIdentifier(lang, resourcePath, loadedTranslation);
+function renderLanguageLoaderAsync(
+  this: LoaderContext,
+  resourcePath: string,
+  loadedTranslation: LoadedTranslation,
+) {
+  return (lang: string) => {
+    const identifier = stringifyRequest(
+      this,
+      createIdentifier(lang, resourcePath, loadedTranslation),
+    );
 
-    return `'${encodeWithinSingleQuotes(
+    return `${JSON.stringify(
       lang,
-    )}': createLanguage(require.resolveWeak('${identifier}'), () => import(
-      /* webpackChunkName: "${getChunkName(lang)}" */
-      '${identifier}'
```

Comment:
```diff
-'./translations.json-en-virtual.json!=!/Users/rmate/work/vocab/packages/virtual-resource-loader/dist/virtual-resource-loader.cjs.js?source=eyJoZWxsbyI6IkhlbGxvIiwid29ybGQiOiJ3b3JsZCJ9!'
+"./translations.json-en-virtual.json!=!../../../../packages/virtual-resource-loader/dist/virtual-resource-loader.cjs.js?source=eyJoZWxsbyI6IkhlbGxvIiwid29ybGQiOiJ3b3JsZCJ9!"

```

Reply:
Done in [`cc66772` (#130)](https://github.com/seek-oss/vocab/pull/130/commits/cc66772711e1f17aed094dca26dd4e01fc74e93e)